src/app/ApplicationTemplate.Mobile/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If i'm not mistaken, we seems to have to androidManifest.xml. witch one is the correct one to be used?

Reply:
Mobile is the correct one, don't look at the Xamarin Android but just Mobile one.